_maps/boxstation.json

```diff
@@ -1,8 +1,5 @@
 {
     "map_name": "Box Station",
     "map_path": "map_files/BoxStation",
-    "map_file": "BoxStation.dmm",
-    "minetype": "lavaland",

```

Comment:
aRe YoU aSsUmInG aLl CoDeBaSeS wIlL uSe LaVaLaNd?

Reply:
if not they can change the code